### PR TITLE
Lots of linting and cleanup for existing pages

### PR DESCRIPTION
**Pull request by jholdstock**
A lot of files changes here but it is largely just linting/formatting changes, updating page titles etc.

The following pages have been reviewed and can be considered ready for #6 
- `blockchain.md`
- `proof-of-work.md`
- `transaction-broadcasting.md`
- `initial-block-download.md`

### CODE REVIEW

File: docs/protocol-p2p-network/initial-block-download.md [hunk 1]
```diff
@@ -17,13 +14,8 @@ downloaded, such as when a previously-caught-up node has been offline
 for a long time. In this case, a node can use the IBD method to download
 all the blocks which were produced since the last time it was online.
 
-Bitcoin Core uses the IBD method any time the last block on its local
+dcrd uses the IBD method any time the last block on its local
```
Comment (jholdstock) [inline docs/protocol-p2p-network/initial-block-download.md]: Confirmed this section in DMs with davec.

File: docs/core-blockchain-concepts/proof-of-work.md [hunk 2]
```diff
@@ -2,80 +2,73 @@
...
-spend to find the hash. 
-
-[NOTE: below is taken basically word for word from Bitcoin docs, and 
-need updating]
+is a measure of the average amount of computational power a miner must
+spend to find the hash.
 
 To keep the network producing blocks on average every 5 minutes, every
-X blocks, the network uses timestamps stored in each block header to 
+144 blocks the network uses timestamps stored in each block header to
```
Comment (jholdstock) [inline docs/core-blockchain-concepts/proof-of-work.md]: Confirmed the values in this section on Matrix: https://matrix.to/#/!HEeJkbPRpAqgAwhXWO:decred.org/$156707443113231treFA:decred.org?via=decred.org&via=matrix.org&via=zettaport.com

**Comment by s-ben** [comment]
I assume it's OK for me to just merge, but let me know if you'd rather I just review w/o approval (what I did on the Projects PR).

**Comment by jholdstock** [comment]
Hm, I think this PR has been closed but not merged

**Comment by s-ben** [comment]
Leaving second review as a comment, as I accidentally screwed up my first one :/ 

On the 'Transaction  Broadcasting' page, 
`If a getdata response message is received, the transaction is sent using tx.` 

What is `tx` exactly here? As a reader that doesn't know already, I want a modifier, something like 'sent using a tx ___'?

On 'Proof-of-Work' page, all looks good. I'm suspicious that some of the text I guessed at made it through, but if it's been through technical review lgtm.

**Comment by jholdstock** [comment]
There are out-standing TODO comments in the PoW page, so it will be revisited as part of #7.
I have added a TODO for checking the tx-broadcasting page because im not sure of the details here, and also for defining `tx`.

**Pull request merged**